### ISSUE TITLE
Failed to login with message "Not Found"

### ISSUE THREAD

**Issue by hotte70** [CiscoCloud/shipped-feedback#25]
Shipped Logon calls Github logon resulting in an API call

https://api.ciscoshipped.io/oauth/github/callback?code=ce26f3b9e1deed5788b9&state=ElRwp

and shows finally

Not Found

Any alternativ way to login to Shipped?

**Comment by chalison** [comment]
I'm trying to use Shipped for the first time and I'm also hitting this issue.

**Comment by mikeihbe** [comment]
There was a cluster outage late last night. It should be back online now.

On Thu, Mar 10, 2016 at 1:37 AM, chalison notifications@github.com wrote:

> I'm trying to use Shipped for the first time and I'm also hitting this
> issue.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CiscoCloud/shipped-feedback/issues/25#issuecomment-194755870
> .

## 

Mike Ihbe
MustWin - Principal

mike@mustwin.com
mikejihbe@gmail.com
skype: mikeihbe
Cell: 651.283.0815

**Comment by chalison** [comment]
I've just tried again, and I still have the same problem. I don't know if it's because I'm a new user or not. I've tried with multiple different browsers and I get the same issue. Any advice would be much appreciated.

**Comment by mikeihbe** [comment]
You're just going to https://ciscoshipped.io and clicking the green "Sign
in with Github" button, right?  We didn't get any downtime alerts last
night, so maybe something else is going on. I have a few other questions:

Did you approve the access we request from github in the application
permissions form (should only happen the first time you log in)? It may
help to remove our access and re-add it.

Have you confirmed your github email address?

Thanks for bearing with us, and hopefully we can get you online!

On Fri, Mar 11, 2016 at 4:31 AM, chalison notifications@github.com wrote:

> I've just tried again, and I still have the same problem. I don't know if
> it's because I'm a new user or not. I've tried with multiple different
> browsers and I get the same issue. Any advice would be much appreciated.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CiscoCloud/shipped-feedback/issues/25#issuecomment-195349128
> .

## 

Mike Ihbe
MustWin - Principal

mike@mustwin.com
mikejihbe@gmail.com
skype: mikeihbe
Cell: 651.283.0815

**Comment by chalison** [comment]
Hi mikeihbe,

I just tried again now and it's working fine. It seems to login OK and I am asked to create my first project. I didn't do anything different now than a few days ago, so I don't know why it's working now, but it is.  :-)   

To answer your questions anyway, yes, I was just clicking the "Sign in with Github" button. The first time I did it I was prompted to allow access, which I did. I double checked in my Github account settings that access was allowed to Shipped. I'm pretty sure I had already confirmed my Github email address.

Regards,

Chris
